production/helm/loki/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: loki
 description: Helm chart for Grafana Loki and Grafana Enterprise Logs supporting both simple, scalable and distributed modes.
 type: application
-appVersion: 3.1.1
-version: 6.14.1
+appVersion: 3.2.0
+version: 6.15.0
```

Comment:
I think you'll need to specify a new version for this helm chart (like 6.16)

Reply:
Good catch 😅